Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "mammut"
-version = "0.11.0"
+version = "0.12.0"
```

Comment:
Can you remove this version bump? I know that it requires a minor version bump but I prefer to only bump when I'm actually making the release.